.github/workflows/conformance.yml

```diff
@@ -0,0 +1,39 @@
+name: conformance
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  everything:
```

Comment:
in the future, when the conformance test suite gets larger, we may want to split this job into several. the job parallelism may reduce the wall clock time of the entire CI pipeline

Reply:
what's the length of time it takes today?

Reply:
The last run took 3:34 minutes, but it can range from 2 to 5